docs/quick_start.en.md

```diff
@@ -14,16 +14,10 @@ Installation for CPU:
 python -m pip install paddlepaddle==3.0.0 -i https://www.paddlepaddle.org.cn/packages/stable/cpu/
 ```
 
-Installation for GPU: Since GPU installation requires specific CUDA versions, the following example is for installing NVIDIA GPU on the Linux platform with CUDA 11.8. For other platforms, please refer to the instructions in the [PaddlePaddle official installation documentation](https://www.paddlepaddle.org.cn/install/quick).
```

Comment:
This paragraph does not need to be deleted.

Reply:
This should be a mistaken deletion. I will add it again.